scripts/common/findAndReplace.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-console */
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const fs = require('fs');
+
+const findAndReplace = (options) => {
+  /* eslint-disable-next-line consistent-return */
+  fs.readFile(options.file, 'utf8', (errMsg, data) => {
+    if (errMsg) {
+      return console.log(errMsg);
```

Comment:
Can we add an `// eslint-disable-next-line no-console` so this isn't appear as a warning when running elsint?